### ISSUE TITLE
[Nautilus 1.0.0.34] BasicText.SetColor does not work until after .ShowMessage is called

### ISSUE THREAD

**Issue by qqqbbb** [SubnauticaModding/Nautilus#582]
code I tested:
```
 BasicText message = new BasicText(); 
 message.SetColor(Color.green);
 message.ShowMessage("Day " );
```

**Comment by JKohlman** [comment]
Yeah that whole file seems very weirdly designed.... None of the properties will really work until you do `ShowMessage` since they all check if TextObject or TextText is null which they are by default:https://github.com/SubnauticaModding/Nautilus/blob/a1b4e2af573cf26cb635e384074b39a0190671fc/Nautilus/Utility/BasicText.cs#L501-L503. They're only set to not null values in `InitializeText` which is only called in `ShowMessage`:https://github.com/SubnauticaModding/Nautilus/blob/a1b4e2af573cf26cb635e384074b39a0190671fc/Nautilus/Utility/BasicText.cs#L459-L464

**Comment by JKohlman** [comment]
As a temporary workaround I think you should be able to do
```cs
BasicText message = new BasicText();
message.ShowMessage("Day " );
message.SetColor(Color.green);
